docs/callouts.md

```diff
@@ -155,54 +156,79 @@ This works like a charm:
 
 > 📸 Cool pix!
 > Vitae reprehenderit at aliquid error voluptates eum dignissimos.
-[block:html]
-{
-  "html": "<style>\n  #my-theme .callout[theme=📸] {\n    --emoji: unset;\n    --icon: \"\";\n  }\n  #my-theme .callout[theme=📷],\n  #my-theme .callout[theme=📸] {\n    --icon-color: #c50a50;\n    --border: var(--icon-color);\n    --title: var(--icon-color);\n  }\n  summary {\n    outline: none;\n    user-select: none;\n  }\n</style>"
+
+<HTMLBlock>{`
+
+<style>
+#my-theme .callout[theme=📸] {
+  --emoji: unset;
+  --icon: \"\";
+}
+#my-theme .callout[theme=📷],
+#my-theme .callout[theme=📸] {
+  --icon-color: #c50a50;
+  --border: var(--icon-color);
+  --title: var(--icon-color);
+}
+summary {
+  outline: none;
+  user-select: none;
 }
-[/block]
+</style>`}
+</HTMLBlock>
+
 </div>
 
-<hr><details><summary><em>Setting the Custom Icon Font</em></summary><br>
+<hr />
+<details>
+  <summary><em>Setting the Custom Icon Font</em></summary><br />
 
 The custom icon font defaults to `FontAwesome`, but you can use any font family available on the page by setting the `--icon-font` variable!
 
 ```css
-.callout[theme=📷] {
+.callout[theme='📷'] {
   --icon-font-family: FontAwesome; /* copied from https://fontawesome.com/v4.7.0/icon/camera-retro */
 }
 ```
 
-</details><hr>
-[block:api-header]
-{
-  "title": "Edge Cases"
-}
-[/block]
-<details><summary>Callouts don't need to have any body text:</summary><br>
+</details>
+<hr />
+
+## Edge Cases
+
+<details>
+  <summary>Callouts don't need to have any body text:</summary><br />
 
 ```
 > 🥇  No body text.
 ```
 
 </details>
 
-> 🥇  No body text.
+> 🥇 No body text.
 
-<details><summary>You can also skip the title, if you're so inclined!</summary><br>
+<details>
+  <summary>You can also skip the title, if you're so inclined!</summary><br />
 
 ```
-> 🥈  
-> 
+> 🥈
+>
 > Lorem ipsum dolor sit amet consectetur adipisicing elit. Error eos animi obcaecati quod repudiandae aliquid nemo veritatis ex, quos delectus minus sit omnis vel dolores libero, recusandae ea dignissimos iure?
```

Comment:
Noticing an edge case here, since callouts don’t actually *require* the extra line break before the body text:

```mdx
> 👍
> The component
```

In the above scenario, RDMD would render the second line of text as the body, whereas in this PR it's being rendered as the header!

Reply:
My original thinking would be to drop support for that? The migration should convert those to including a new line.

But if you think it's worth keeping, I can add some extra bits.

Reply:
https://linear.app/readme-io/issue/RM-9881/callout-syntax-regression

Reply:
That's fair; I'm down to drop support, especially if the migration will catch and convert these! Will we that happen automatically, or will we need to manually convert ‘em?

Reply:
The migration is basically:
```
const newDoc = rmdx.mdx(rdmd.mdast(doc))
```

As long as RDMD can read it correctly, and RMDX can write it correctly, it should just work :tm:

Reply:
Ahhhhh, right of course, I could've imagined that! And then there are the couple of cases where it can't read the old syntax, but *those* should be covered by RM-9839. Got it.

Reply:
Let's leave RM-9881 open and we can discuss further whenever we get to those fast follows.

Reply:
Thanks for the context/details here @kellyjosephprice!